platforms/gke/base/core/gke_enterprise/policycontroller/feature.tf

```diff
@@ -29,10 +29,6 @@ resource "google_gke_hub_feature_membership" "cluster_spolicycontroller" {
   project    = data.google_project.default.project_id
 
   policycontroller {
-    #enabled                    = true
-    #referential_rules_enabled  = true
-    #template_library_installed = true
-
     policy_controller_hub_config {
```

Comment:
Can we please enable, or add variables to enable the following:

- `mutation_enabled`
- `log_denies_enabled`
- `referential_rules_enabled`

Ref: https://registry.terraform.io/providers/hashicorp/google/latest/docs/resources/gke_hub_feature_membership#nested_policy_controller_hub_config